strings/tips.txt

```diff
@@ -188,14 +218,6 @@ As a Cultist, your team starts off very weak, but if necessary can quickly conve
 As a Cultist, the Blood Boil rune will deal massive amounts of brute damage to non-cultists, stamina damage to Ratvarian scum, and some damage to fellow cultists of Nar'Sie nearby, but will create a fire where the rune stands on use.
 As a Cultist, you can create an army of manifested goons using a combination of the Manifest rune, which creates homunculi from ghosts, and the Blood Drain rune, which drains life from anyone standing on any blood drain rune.
 As a Cultist or Servant, check the alert in the upper-right of your screen for all the details about your cult's current status and objective.
```

Comment:
Remove the "or Servant" from this line too, as well as any other lines that might mention servant.